cmd/util/ledger/migrations/cadence_value_diff.go

```diff
@@ -383,15 +360,6 @@ func (dr *CadenceValueDiffReporter) diffDomain(
 			)
 		}
 	}
-
-	log.Info().
-		Msgf(
-			"Finished diffing %x storage domain containing %d elements (%d payloads) in %s",
-			dr.address[:],
-			len(sharedKeys),
-			oldRuntime.PayloadCount,
-			time.Since(startTime),
-		)
```

Comment:
Might be still useful

Reply:
Good idea!  Done in PR #7016 because this PR got auto-merged.